test/base/test_buffer.py

```diff
@@ -278,7 +280,79 @@ def test_pickle():
                        pbuf.weight[np.arange(len(pbuf))])
 
 
+def test_hdf5():
+    size = 100
+    vbuf = ReplayBuffer(size, stack_num=2)
+    lbuf = ListReplayBuffer()
+    pbuf = PrioritizedReplayBuffer(size, 0.6, 0.4)
+    device = 'cuda' if torch.cuda.is_available() else 'cpu'
+    rew = torch.tensor([1.]).to(device)
+    for i in range(4):
+        vbuf.add(
+            obs=Batch(index=np.array([i])),
+            act=i,
+            rew=rew,
+            done=0,
+            info={"number": {"n": i}},
+        )
+        lbuf.add(
+            obs=Batch(index=np.array([i])),
+            act=i,
+            rew=rew,
+            done=0,
+            info={"number": {"n": i}},
+        )
+        pbuf.add(
+            obs=Batch(index=np.array([i])),
+            act=i,
+            rew=rew,
+            done=0,
+            info={"number": {"n": i}},
+            weight=np.random.rand()
+        )
+    buffers = {
+            "v": vbuf,
```

Comment:
I know it is just a unit test, but could you make those fields a little bit more explicit ?

Reply:
Ok, I will go over the test again once I have given your suggestions for simplifying the code a try.